app/javascript/stylesheets/button.css

```diff
@@ -1,68 +1,98 @@
 .button {
-  padding: 0.5rem 0.75rem 0.5rem 0.75rem;
```

Comment:
Everything from here on out is CSS changes. I'm happy to walk through them if you think that would be helpful, but the goal in all of the changes was to make the site look nice not just on a large screen, but on mobile and every screen size in between.